main.py

```diff
@@ -263,7 +262,7 @@ def print_report():
     # Fetch required fields, including total_cost and total_tokens
     c.execute(
         """
-        SELECT file_name, execution_status, time_taken, total_embedding_cost, total_embedding_tokens, total_llm_cost, total_llm_tokens, error_message
+        SELECT file_name, execution_status, time_taken, total_embedding_cost, total_embedding_tokens, total_llm_cost, total_llm_tokens
```

Comment:
@pk-zipstack small suggestion. Instead of removing the error all together can we wrap and truncate? Having some error message might be helpful for quick debug.

Reply:
@ritwik-g By truncate you mean to cut the error message? For example `"Error while processing file: /home/praveen/Downloads/test_data/wordpress-pdf-invoice-plugin-sample.pdf: the JSON object must be str, bytes or bytearray, not dict"`, 

can be `"Error while processing file: /home/praveen/Downloads/test_data/wordpress-pdf-invoice-plugin-sample.pdf:..."`

Reply:
Yes we can find the length which will fit in the column well and use that to truncate.

Reply:
Sure, I'll get it done @ritwik-g